### PR TITLE
Update cypress version to latest version

### PR DESCRIPTION
**Pull request by Jackthomsonn**
Bringing the docker image inline with local versions

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,14 @@
-FROM cypress/included:9.2.0
+FROM cypress/included:9.5.3
+
+# Update package lists
+apt-get install update -y
```
Comment (benjefferies) [inline Dockerfile]: RUN apt-get install update -y && apt-get curl -y

Comment (Jackthomsonn) [reply Dockerfile]: Your wish is my command 🧞

File: Dockerfile [hunk 2]
```diff
@@ -1,9 +1,11 @@
-FROM cypress/included:9.2.0
+FROM cypress/included:9.5.3
+
+# Update package lists and install curl
+apt-get install update -y && apt-get curl -y
```
Comment (benjefferies) [inline Dockerfile]:
```suggestion
RUN apt-get install update -y && apt-get curl -y
```

File: Dockerfile [hunk 3]
```diff
@@ -1,9 +1,11 @@
-FROM cypress/included:9.2.0
+FROM cypress/included:9.5.3
```
Comment (benjefferies) [inline Dockerfile:1]:
```suggestion
FROM cypress/included:9.5.3

# Install gcloud
RUN apt-get -y update && apt-get install -y curl gnupg && \
  echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
  curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg  add - && \
  apt-get update -y && \
  apt-get install -y google-cloud-sdk libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb

# Install SOPS
RUN apt-get install -y git && \
  curl -Lo /usr/local/bin/sops https://github.com/mozilla/sops/releases/download/v3.7.1/sops-v3.7.1.linux && chmod +x /usr/local/bin/sops && \
  git clone https://github.com/nodenv/nodenv.git ~/.nodenv && ln -s $HOME/.nodenv/bin/nodenv /usr/local/bin/nodenv && \
  mkdir -p "$(nodenv root)"/plugins && git clone https://github.com/nodenv/node-build.git "$(nodenv root)"/plugins/node-build

# Install chrome
RUN apt-get -y install fonts-liberation xdg-utils && \
  wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb && \
  dpkg -i google-chrome-stable_current_amd64.deb && \
  rm google-chrome-stable_current_amd64.deb

ENV LANG=C.UTF-8 PATH=/root/.asdf/bin:/root/.asdf/shims:$PATH

RUN git clone https://github.com/asdf-vm/asdf.git $HOME/.asdf --branch v0.9.0 && \
  echo . $HOME/.asdf/asdf.sh >> $HOME/.bashrc && \
  echo . $HOME/.asdf/completions/asdf.bash >> $HOME/.bashrc && \
  asdf plugin add nodejs https://github.com/asdf-vm/asdf-nodejs.git && \
  echo legacy_version_file = yes > $HOME/.asdfrc

ENTRYPOINT []
```

**Pull request merged**